tests/autodiff/material2/MxLayeredMaterial.slang

```diff
@@ -55,8 +55,7 @@ public struct MxLayeredMaterial : IMaterial
     public UsedMaterialInstance setupMaterialInstance(out MaterialInstanceData miData)
     {
         float3 albedo = getAlbedo(baseColor);
-        UsedMaterialInstance mi;
         for (uint i = 0; i < 3; i++) miData.data[i] = albedo[i];
-        return mi;
+        return UsedMaterialInstance();
```

Comment:
This is not needed, struct's automatically call their default constructor if available. This may need to be a case when checking for uninitialized variables.